src/commands/lambda/README.md

```diff
@@ -146,7 +146,7 @@ Instead of supplying arguments, you can create a configuration file in your proj
 }
 ```
 
-## Troubleshooting Serverless Instrumentation
+## Troubleshooting Lambda Instrumentation
```

Comment:
Optional change. Since we have documentation for both cloud-run and lambda, I figured we could be more specific here